uvloop/loop.pxd

```diff
@@ -1,15 +1,35 @@
 # cython: language_level=3
 
+cdef extern from *:
+    '''
+    enum { UV_STREAM_RECV_BUF_SIZE = 256000,
+           SSL_READ_MAX_SIZE = 256 * 1024,  // 250kb
+          };
+    const float SSL_HANDSHAKE_TIMEOUT = 60.0; // Number of seconds to wait for SSL handshake to complete The default timeout matches that of Nginx.
+    const float SSL_SHUTDOWN_TIMEOUT  = 30.0; // Number of seconds to wait for SSL shutdown to complete The default timeout mimics lingering_time
+    '''
+    const bint UV_STREAM_RECV_BUF_SIZE
+    const bint SSL_READ_MAX_SIZE
+
+    const float SSL_HANDSHAKE_TIMEOUT
```

Comment:
maybe `#define` instead of `const float`?

Reply:
not sure it would work, I only use #defines when the language does not support it